remoteit/desktop
# [DESK-851] fix uninstall

### Description

https://remoteit.atlassian.net/browse/DESK-851 video on the comment 

[DESK-851] fix uninstall


### Test plan

1. Sign in using Windows
2. Go to Setting
3. Click on Uninstall
4. You should see  remove service correctly

### Ticket

https://remoteit.atlassian.net/browse/DESK-851


[DESK-851]: https://remoteit.atlassian.net/browse/DESK-851
